scripts/release-tools/clusters.sh

```diff
@@ -212,6 +229,62 @@ create_qa_gke_cluster() {
   kubectl -n stackrox get pods
 }
 
+create_long_running_cluster() {
+
+  CLUSTER_NAME="$(get_cluster_name long-running)"
+  export CLUSTER_NAME
+
+  echo "cluster_name= $CLUSTER_NAME"
+  
+  status="$(check_cluster_status "$CLUSTER_NAME")"
+  
+  if does_cluster_exist "$CLUSTER_NAME"; then
+      echo "Unable to create cluster"
+  else
+      infractl create gke-default $CLUSTER_NAME --lifespan 168h --arg nodes=5 --wait --slack-me
+  fi
+  
+  # Set your local kubectl context to the remote cluster once the above completes successfully.
+  infractl get $CLUSTER_NAME --json | jq '.Connect' -r | bash
+
+  kubectl -n stackrox get pod
+  
+  
+  export MAIN_IMAGE_TAG=$(git describe --tags --abbrev=0) # Release version, e.g. 3.63.0-rc.2.
+  export API_ENDPOINT="localhost:8000"
+  
+  export STORAGE=pvc # Backing storage
+  export STORAGE_CLASS=faster # Runs on an SSD type
+  export STORAGE_SIZE=100 # 100G
+  export MONITORING_SUPPORT=true # Runs monitoring
+  export LOAD_BALANCER=lb
+  
+  toplevel_dir="$(git rev-parse --show-toplevel)"
+  "$toplevel_dir/deploy/k8s/central.sh" # Launches central
+  wait_for_pods_to_be_ready
+  
+  # Open port-forward to central, e.g. with
+  kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
+  sleep 60
+  
+  export ROX_ADMIN_USERNAME=admin
+  
+  export ROX_ADMIN_PASSWORD="$(cat deploy/k8s/central-deploy/password)"
+  export ROX_PASSWORD="$ROX_ADMIN_PASSWORD"
+
+  "$toplevel_dir/deploy/k8s/sensor.sh"
+  wait_for_pods_to_be_ready
+
+  kubectl -n stackrox set env deploy/sensor MUTEX_WATCHDOG_TIMEOUT_SECS=0
+  kubectl -n stackrox set env deploy/sensor ROX_FAKE_KUBERNETES_WORKLOAD=long-running
+  wait_for_pods_to_be_ready
```

Comment:
Do we need to wait for pods twice here?

Reply:
Yes, I think this is necessary, as deploying the long-running workload changes several things in the cluster - including restarts of some pods.